README.md

```
@@ -26,45 +26,68 @@ Before running the project, ensure that you have the following installed:
+Ensure Docker is installed and running on your machine.
 
-   services:
-     rabbitmq:
-       image: rabbitmq:3.13-management
-       container_name: rabbitmq
-       ports:
-         - '5672:5672'
-         - '15672:15672'
+1. Navigate to the project directory.
+2. Start RabbitMQ Container using below command
+   ```sh
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)